lib/generators/class_variants/install/templates/class_variants.rb.tt

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# ClassVariants.configure do |config|
+#   # allow to post process classes with an external utility like TailwindMerger
+#   config.process_classes_with do |classes|
+#     TailwindMerge::Merger.new.merge(classes)
+#   end
+# end
```

Comment:
I was thinking about generating the initializer uncommented and leave only the `process_classes_with` option commented out, but I think I like this approach better.
We are not configuring anything with it, we might as well not load it.
Good call!